rusregistry/entities/entity.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import random
+import string
+from typing import List, Dict
+import followthemoney as ftm
+
+def get_unique_id():
+    # This should be fixed
+    return random.sample(string.ascii_letters, 24)
+
+class Entity:
+    def __init__(self, item):
+        self.entity_type = None
+        if type(item) == pd.core.series.Series:
+            self.from_csv_row(item.copy())
+        else:
+            raise TypeError("Unrecognised input type! Currently supported types: [pd.Series]")
+
+    def from_csv_row(self, row: pd.core.series.Series):
+        self.data_dict = row.to_dict()
+
+    def make_id(self, entity):
+        entity.make_id(self.data_dict.get('id', get_unique_id()))
+        return entity
```

Comment:
This is the most important to me: Is there any chance we can make natural IDs like `ru-ogrn-XXXXX` for the companies? That way it becomes really easy to store and manage de-dupe info!